platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/schedulers/builders/TaskSchedulerBuilder.java

```diff
@@ -86,6 +91,7 @@ public TaskSchedulerBuilder(@NonNull final StandardWiringModel model, @NonNull f
             throw new IllegalArgumentException("TaskScheduler name must not be empty");
         }
         this.name = name;
+        this.pool = Objects.requireNonNull(defaultPool);
```

Comment:
Should `ForkJoinPool.commonPool()` still be used to initialize `this.pool` above? Looks like it's always overwritten here.